google-http-client/src/test/java/com/google/api/client/http/MultipartContentTest.java

```diff
@@ -26,55 +26,76 @@
  */
 public class MultipartContentTest extends TestCase {
 
+  private static final String BOUNDARY = "__END_OF_PART__";
   private static final String CRLF = "\r\n";
   private static final String CONTENT_TYPE = Json.MEDIA_TYPE;
-  private static final String HEADERS =
-      "Content-Length: 3"
-          + CRLF
-          + "Content-Type: application/json; charset=UTF-8"
-          + CRLF
-          + "content-transfer-encoding: binary"
-          + CRLF;
+  private static final String HEADERS = headers("application/json; charset=UTF-8", "foo");
+
+  private static String headers(String contentType, String value) {
+      return "Content-Length: " + value.length() + CRLF
+          + "Content-Type: " + contentType + CRLF
+          + "content-transfer-encoding: binary" + CRLF;
+  }
+
+  public void testRandomContent() throws Exception {
+    MultipartContent content = new MultipartContent();
+    String boundaryString = content.getBoundary();
+    assertNotNull(boundaryString);
+    assertTrue(boundaryString.startsWith(BOUNDARY));
+    assertTrue(boundaryString.endsWith("__"));
+    assertEquals("multipart/related; boundary=" + boundaryString, content.getType());
+
+    final String[][] VALUES = new String[][] {
+            {"Hello world", "text/plain"},
+            {"<xml>Hi</xml>", "application/xml"},
+            {"{x:1,y:2}", "application/json"}
+    };
+    StringBuilder expectedContentSB = new StringBuilder();
```

Comment:
no abbreviations per google style, but simply "expectedContent" or even just expected is fine.

Reply:
expectedContent is used later. renamed to expectedStringBuilder